src/QuickMarcEditor/QuickMarcEditor.test.js

```diff
@@ -149,8 +150,25 @@ describe('Given QuickMarcEditor', () => {
     });
   });
 
+  describe('when clicked save and keep editing button', () => {
```

Comment:
Please also add a test with `action` `QUICK_MARC_ACTIONS.CREATE` or `QUICK_MARC_ACTIONS.DUPLICATE` to check that the `Save & Keep editing` button is not shown